docs/source/contributing/developing.adoc

```diff
@@ -111,18 +111,18 @@ $ glide install -v
 ----
 
 [[update-dependencies]]
-=== Updating dependencies
+=== Updating Dependencies
 
 If your work requires a change to the dependencies, you need to update the Glide configuration.
 
 . Edit *_glide.yaml_* to change the dependencies as needed.
 
-. Delete *_glide.lock_*  and re-create the vendor directory by running `make vendor`.
+. Delete *_glide.lock_* and re-create the vendor directory by running `make vendor`.
  Glide will recognize that there is no lock file and recalculate the required dependencies.
 
-. Check-in the updated *_glide.yaml_* and *_glide.lock_*.
+. Check-in the updated *_glide.yaml_* and *_glide.lock_* files.
```

Comment:
_files_ is imo superfluous here. It just makes it sounds odd, even though it is correct. I am saying "I check-in start.go" not "I am checking in the start.go file". That's implicit.